server/secrets.go

```diff
@@ -34,7 +34,7 @@ func readMountFile(mountFilePath string) ([]string, error) {
 	var mountPaths []string
 	file, err := os.Open(mountFilePath)
 	if err != nil {
-		logrus.Warnf("file doesn't exist %q", mountFilePath)
+		logrus.Debugf("file doesn't exist %q, skipping...", mountFilePath)
```

Comment:
We should keep this as a warning I think.

Reply:
at the cost of cluttering logs with this if people doesn't have this file?

Reply:
Yes I would keep this as a warning.  We are going to package it with the file, so it should be weird if our distributions did not have it.